backend-agent/cli.py

```diff
@@ -379,6 +379,62 @@ def run(args):
               file=sys.stderr)
 
 
+def run_all_logic(args):
+    """Run all LLM attacks with specified target and evaluation models."""
+    # Load JSON spec
+    if not args.file:
+        print(
+            'No file given. Enter specification using stdin.',
+            file=sys.stderr
+        )
```

Comment:
the file should be the one in data/all/default.json, but it seems you're not loading it here